tests/functional/strategy/test_startup.py

```diff
@@ -1,79 +1,101 @@
 def test_startup(token, gov, vault, strategy, keeper, chain):
-    # Strategy has no debt or anything yet until, we call harvest
-    assert token.balanceOf(strategy) == 0
+    all_strategies = [strategy] + ["0x0000000000000000000000000000000000000000"] * 39
+    er = lambda: vault.expectedReturn(strategy)
```

Comment:
A bit nitpicky but, `expectedReturn` for the lambda seems good as a name, since we are calling it way below and may need context without scrolling up